core/src/main/java/ai/timefold/solver/core/impl/localsearch/decider/acceptor/AcceptorFactory.java

```diff
@@ -45,6 +47,7 @@ public Acceptor<Solution_> buildAcceptor(HeuristicConfigPolicy<Solution_> config
                 buildMoveTabuAcceptor(configPolicy),
                 buildSimulatedAnnealingAcceptor(configPolicy),
                 buildLateAcceptanceAcceptor(),
+                buildDiversifiedLateAcceptanceAcceptor(configPolicy),
```

Comment:
Isn't it strange that one doesn't require a config policy and the other one does?